lib/detect/darwin.js

```diff
@@ -2,14 +2,11 @@
 
 const path = require('path')
 const plist = require('simple-plist')
-const osxFindExecutable = require('@httptoolkit/osx-find-executable')
+const { findOsxExecutable } = require('@httptoolkit/osx-find-executable')
 
 function finder (bundleId, versionKey) {
   return function find (callback) {
-    osxFindExecutable(bundleId, function (err, execPath) {
-      // Ignore not found error
-      if (err) return callback()
-
+    findOsxExecutable(bundleId).then(function (execPath) {
```

Comment:
It would be good to escape the promise chain when converting from promises to callbacks.

Use the [`catering`](https://www.npmjs.com/package/catering) utility for that:

```js
const { fromPromise } = require('catering')
```

```js
fromPromise(findOsxExecutable(bundleId), function (err, execPath) {
  // Ignore not found error
  // ..
})
```